frontend/app/projects/template.hbs

```diff
@@ -128,6 +134,7 @@
             </thead>
             <tbody>
               {{#each this.tasks as |task|}}
+                {{#if (or (and this.hideArchivedTasks (not task.archived)) (not this.hideArchivedTasks))}}
```

Comment:
```suggestion
                {{#unless (and this.hideArchivedTasks task.archived)}}
```
Seems more readable to me, what ya think? Or will this trigger `simple-unless` template lint rule? :thinking: